lian/src/lian_nav2/src/lian_planner.cpp

```diff
@@ -20,14 +23,45 @@ void LianPlaner::configure(
   tf_ = tf;
   planner_->costmap_ = costmap_ros->getCostmap();
   global_frame_ = costmap_ros->getGlobalFrameID();
-
+  
 }
 
-nav_msgs::msg::Path ThetaStarPlanner::createPlan(
+nav_msgs::msg::Path LianPlaner::createPlan(
   const geometry_msgs::msg::PoseStamped & start,
   const geometry_msgs::msg::PoseStamped & goal)
 {
-  nav_msgs::msg::Path global_path;
-  auto start_time = std::chrono::steady_clock::now();
   
+  searchParams[CNS_TAG_SX] = start.pose.position.x
+  searchParams[CNS_TAG_SY] = start.pose.position.y
+  searchParams[CNS_TAG_FX] = goal.pose.position.x
+  searchParams[CNS_TAG_FY] = goal.pose.position.y
+
+  nav_msgs::msg::Path lian_path;
+  auto start_time = std::chrono::steady_clock::now();
+
+  if (sr.pathfound) {
```

Comment:
where does `sr` come from?

Reply:
sr for search result come from the mission class, I included mission.h